index.bs

```diff
@@ -1252,13 +1252,12 @@ the interaction with garbage collection.
 
 <h2 id="api-surface">JavaScript API Surface Concerns</h2>
 
-<h3 id="attributes-like-data">Attributes should behave like data properties</h3>
+<h3 id="accessors-like-data" oldids="attributes-like-data">Accessors should behave like data properties</h3>
 
-[[!WEBIDL]] attributes should act like simple JavaScript object properties.
+JavaScript accessors (specified as [[!WEBIDL]] attributes with separate getter and setter methods in Web APIs)
```

Comment:
I think the original sentence is clearer.

Reply:
```suggestion
Any syntax that looks like JavaScript properties (such as accessors or [[!WEBIDL]] attributes)
```

@cynthia @hober what about that?